remo/reports/tests/test_api.py

```diff
@@ -69,3 +73,104 @@ def test_get_remo_url(self):
         data = ActivitiesDetailedSerializer(
             report, context={'request': request}).data
         ok_(report.get_absolute_url() in data['remo_url'])
+
+
+class TestActivitiesKPIView(RemoTestCase):
```

Comment:
Probably it's a good idea to add a couple of POST requests and capture the return code, eg: `405 Method Not Allowed`.